src/gluonts/support/util.py

```diff
@@ -390,11 +390,12 @@ def cumsum(
     return cumulative_sum.squeeze(axis=exp_dim)
 
 
-def weighted_average(
+def masked_weighted_average(
```

Comment:
This function computes the weighted average, so its name can stay `weighted_average` :-)

Changing name would be a breaking change, so let’s avoid it unless it’s necessary

Reply:
Ok, fine... :(((